### PR TITLE
fix(enterprise-cloud-database.create): add missing label and contracts

### PR DESCRIPTION
**Pull request by Jisay**
# :house_with_garden: Internal
ref:
* DTRSD-8137

### CODE REVIEW

File: packages/manager/modules/enterprise-cloud-database/src/create/create.html [hunk 1]
```diff
@@ -366,13 +366,14 @@ <h5>
...
                 </enterprise-cloud-database-payment-info-component>
                 <div class="row">
-                    <div
-                        class="col-md-6"
-                        data-ng-if="$ctrl.order"
-                        data-ovh-contracts="$ctrl.order.contracts"
-                        data-full-text="false"
-                        data-ovh-contracts-validated="$ctrl.contractsAccepted"
-                    ></div>
+                    <div class="col-md-6" data-ng-if="$ctrl.order">
```
Comment (FredericEspiau) [inline packages/manager/modules/enterprise-cloud-database/src/create/create.html]: Hm if `$ctrl.order` is null, then the `class="row"` right above will be incorrect so you can remove the row altogether

File: packages/manager/modules/enterprise-cloud-database/src/payment-info/payment-info.html [hunk 2]
```diff
@@ -2,14 +2,15 @@
...
-            data-ng-src="{{::$ctrl.defaultPaymentMethod.original.icon.data}}"
-            alt=""
-        />
+        <img data-ng-src="{{::$ctrl.defaultPaymentMethod.icon.data}}" alt="" />
         <span data-ng-bind="$ctrl.defaultPaymentMethod.label"></span>
-        <span class="text-success"
-            >[{{ ::$ctrl.defaultPaymentMethod.paymentType.text }}]</span
-        >
+        <span class="text-success">
+            <span>[</span>
```
Comment (FredericEspiau) [inline packages/manager/modules/enterprise-cloud-database/src/payment-info/payment-info.html]:
```suggestion
            [
```
should be enough ;)

File: packages/manager/modules/enterprise-cloud-database/src/create/create.html (hunk 1 above)
Comment (antleblanc) [reply packages/manager/modules/enterprise-cloud-database/src/create/create.html]: You're right! We will improve it.

File: packages/manager/modules/enterprise-cloud-database/src/payment-info/payment-info.html (hunk 2 above)
Comment (antleblanc) [reply packages/manager/modules/enterprise-cloud-database/src/payment-info/payment-info.html]: Correct!

**Pull request merged**